ast/desugar/Desugar.cc

```diff
@@ -26,13 +26,14 @@ struct DesugarContext final {
     core::NameRef enclosingMethodName;
     bool inAnyBlock;
     bool inModule;
+    bool calledFromExtractToVariable;
```

Comment:
I'm generally -1 on naming things "calledFromX"

The option should describe what behavior is being done. So like "preserveConcreteSyntax" or something like that.